CHANGELOG.md

```diff
@@ -109,6 +109,11 @@ Bottom level categories:
 - Breaking change: [`wgpu_core::pipeline::ProgrammableStageDescriptor`](https://docs.rs/wgpu-core/latest/wgpu_core/pipeline/struct.ProgrammableStageDescriptor.html#structfield.entry_point) is now optional. By @ErichDonGubler in [#5305](https://github.com/gfx-rs/wgpu/pull/5305).
 - `Features::downlevel{_webgl2,}_features` was made const by @MultisampledNight in [#5343](https://github.com/gfx-rs/wgpu/pull/5343)
 
+#### WebGPU
+
+- Implement the `device_set_device_lost_callback` method for `ContextWebGpu`. By @suti in [#5438](https://github.com/gfx-rs/wgpu/pull/5438)
```

Comment:
oh actually there's a new WebGPU section now further down because I merged another PR that added it just now. Can you please merge the two? thanks!

Reply:
The robot reminds me of the branch differences. I have clicked rebase and it looks like it should be successful.

Reply:
yeah the merge conflict is resolved, but the changelog now has two `#### WebGPU` sections where there should be just one

Reply:
there https://github.com/gfx-rs/wgpu/pull/5438/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edL122

Reply:
done